tier2_standards/release_procedure.md

```diff
@@ -0,0 +1,129 @@
+# Project Must Define and Document a Consistent Release Procedure
+
+This is one of the tier 2 standards. See full list [on the overview page](README.md).
+
+## Short description
+
+We require that teams adopt a consistent procedure for making a new 'release' and document (or reference an existing document of) said procedure somewhere within the repository and/or project documentation.  In this context, we define a 'release' as the tagging of a new software version and uploading that version of the software to either PyPI or Conda (as was part in the [Tier 1 standards](https://github.com/spacetelescope/ins-jwst-community-software/blob/master/tier1_standards/conda_or_pip.md)).
+
+## Importance of this standard
+
+Adopting a consistent, documented release procedure has several benefits:
+
+- It helps to reduce the learning curve for performing releases, and thus helps to enable any current or future team member to perform a release.
+- It makes it clear to developers and users which distribution source (i.e. `pypi` or `conda`) is used for releases.
+- By maintaining release notes, it makes it clear to developers and users which features and/or bug fixes are part of specific versions of the software. (For more information, see the [Release Notes Tier 2 standard](release_notes.md))
```

Comment:
I’m not sure about the 3rd bullet here. It seems more related to why you should write release notes compared to why you should document how to do a release.

Reply:
Good point.  Perhaps I can change this to something like "It serves as an explicit reminder to update/maintain release notes and tagged versions of the software"